src/components/capture/create/Store.ts

```diff
@@ -4,13 +4,6 @@ import { isEqual } from 'lodash';
 import { devtoolsInNonProd } from 'utils/store-utils';
```

Comment:
There are anonymous actions defined in this file. For the ease of debugging, each action should be accompanied by a descriptive string.

Reply:
I do not know what you're talking about.

Reply:
There are days I wish GitHub adopted some of the functionality of Azure Repos. I am unable to leave a comment anywhere in a given file, so it is difficult to highlight a proper example. `set` can have three arguments, the third argument dictates the string that appears in Redux Dev Tools. When that third argument is undefined, a vague `anonymous` appears in the dev tools. Currently, every `hasChanges` call is ambiguous in the tool.